draft-ietf-sframe-enc.md

```diff
@@ -429,29 +429,33 @@ header ----+------------------>| AAD
 |               |      |               |     |               |
 +---------------+      +---------------+     +---------------+
 ~~~~~
-{: title="Encryption flow" }
+{: title="Encryption flow with per-frame encryption" }
 
 ### Decryption
 
-The receiving clients buffer all packets that belongs to the same frame using the frame beginning and ending marks in the generic RTP frame header extension, and once all packets are available, it passes it to SFrame for decryption.  The KID field in the SFrame header is used to find the right key for the encrypted frame.
+Before decrypting, a client needs to assemble a full SFrame ciphertext.  When
+SFrame is applied per-packet, this is done by extracting the payload of a
```

Comment:
I think that the mentions about specific usage per frame and per packet should be moved to a diferent document. However, i am ok to keep the text here as a `Note:`

Reply:
Agreed with the direction to move per frame/packet to another document if possible.
That said, let's do that as as a follow-up.